src/components/RichTextEditor.js

```diff
@@ -125,8 +125,18 @@ export default class RichTextEditor extends React.PureComponent {
     } = this.props
 
     Promise.resolve(window.tinymce || loadScript(TINYMCE_ASSET_URL))
+    .then(() => new Promise(resolve => {
+      const mutex = setInterval(() => {
+        if (window.RichTextEditorMutex) return
```

Comment:
let's add a comment about what this is for, e.g. "When multiple RichTextEditor components are mounted at once, we have to serialize their initialization, or weird things happen."